C7GameData/AIData/TilePath.cs

```diff
@@ -29,6 +29,32 @@ public int PathLength() {
 			return path != null ? path.Count : -1;
 		}
 
+		public int PathCost(Tile from, float perTurnMovePoints, float remainingMovementPoints) {
```

Comment:
This should round up, as partial movement still uses a turn. In Civ3 "0" is only shown when movement is completely free.

Reply:
Partially done. In my git noob-ness, I accidentally ended up with the change applied to https://github.com/C7-Game/Prototype/pull/464/ :(

I'm using to `hg update <this PR>; hg amend; hg evolve; hg update tip; hg uploadchain` to do something like this.

My `git checkout <this PR>; git switch -c <temp branch for this fix>; git switch <child PR>;  git merge <temp branch for this fix>` didn't work. Maybe I wanted `rebase` or something.

Let me know if you want me to mess with this more to get the fix in this PR, sorry about that.

Reply:
No problem. I'm reviewing and merging in chronological order to keep the history clean so I hadn't seen the rest of the changes yet.
I don't think there's a simple answer for when you're queuing up multiple linear merges. Well, other than to do it all in one PR, but that's harder to review.